test/e2e/framework/natscluster.go

```diff
@@ -194,7 +206,10 @@ func (f *Framework) PatchCluster(natsCluster *natsv1alpha2.NatsCluster) (*natsv1
 // VersionForPod returns the version of NATS reported by the specified pod.
 func (f *Framework) VersionForPod(pod v1.Pod) (string, error) {
 	// Check the "/varz" endpoint for the number of established routes.
-	r, err := http.Get(fmt.Sprintf("http://%s:%d/varz", pod.Status.PodIP, constants.MonitoringPort))
+	client := &http.Client{
```

Comment:
Do we need to worry about https here?

Reply:
Yes, this test is not using https for this check but if https monitoring is enabled then this has to be https.